Tutorials/Sprite-Related-Tutorials/Getting Started With Sprites in Splashkit Tutorial - C#/Getting Started With Sprites in Splashkit Tutorial - CSharp.mdx

```diff
@@ -281,14 +431,48 @@ public class Program{
       }
 }
 ```
-
   </TabItem>
-<TabItem label="C# 'pure'">
+    <TabItem label="C# 'pure'">
   **Coming soon**
   </TabItem>
 
-<TabItem label="C++">
-  **Coming soon, requires full tutorial re-write**
+  <TabItem label="C++">
+```cpp
+#include "splashkit.h"
+
+int main()
+{
+    // Instantiation of Bitmap Object
+    bitmap HappySun = load_bitmap("Happy Sun", "sprite01.png");
+
+    // Instantiation of Sprite Object(s)
+    sprite HappySunSprite = create_sprite("Happy Sun", HappySun);
+
+    // Creation of Window
+    open_window("Learning Sprites!", 800, 800);
+
+    // Set Sprite position, then draw it to the window
+    move_sprite_to(HappySunSprite, 400, 400);
+    draw_sprite(HappySunSprite);
+
+    // Update the screen and hold for 5 seconds
+    refresh_screen();
+    delay(5000);
+
+    for (int i = 0; i < 150; i++)
+{
+    clear_window(current_window(), COLOR_WHITE);
+    move_sprite_to(HappySunSprite, (sprite_x(HappySunSprite)), (sprite_y(HappySunSprite)));
+
+    draw_sprite(HappySunSprite);
+
+    refresh_screen();
+    delay(10);
+}
```

Comment:
The formatting here needs to be slightly fixed (i.e. tabs added before the curly braces in this for loop). You'll likely want to make changes mentioned earlier here too.